hq/app/AppComponents.scala

```diff
@@ -24,13 +28,37 @@ class AppComponents(context: Context)
     csrfFilter,
     new HstsFilter()
   )
+  implicit val awsClient: AWSSimpleSystemsManagement = AWSSimpleSystemsManagementFactory("eu-west-1", "security")
+
+  val configraun: Configuration = {
+
+    configuration.getOptional[String]("stage") match {
+      case Some("DEV") =>
```

Comment:
It looks like this means we all have to use the exact same config in DEV. That means we can't (for example) have different accounts configured to match the credentials we have available. It also means we can't try things by changing config without it affecting others.

Reply:
It means the DEV config _for objects fetched via Configraun_ will be universal, yes.

Currently that is only the Snyk token and Organisation name.